### ISSUE TITLE
[Dev] Load train.csv to Huggingface Datasets `Dataset` 

### ISSUE THREAD

**Issue by KimDaeUng** [boostcampaitech2/klue-level2-nlp-14#8]
## Coment
- 명훈님 베이스라인 코드형태로 entity를 처리할 때 필요한 입력 데이터셋입니다.
- 이전에 공유드렸던 코드에서 누락된 `dataset['label'] = dataset['label'].map(label2id)
`을 추가하였습니다.

## Code
```python
class_names = ['no_relation', 'org:top_members/employees', 'org:members',
       'org:product', 'per:title', 'org:alternate_names',
       'per:employee_of', 'org:place_of_headquarters', 'per:product',
       'org:number_of_employees/members', 'per:children',
       'per:place_of_residence', 'per:alternate_names',
       'per:other_family', 'per:colleagues', 'per:origin', 'per:siblings',
       'per:spouse', 'org:founded', 'org:political/religious_affiliation',
       'org:member_of', 'per:parents', 'org:dissolved',
       'per:schools_attended', 'per:date_of_death', 'per:date_of_birth',
       'per:place_of_birth', 'per:place_of_death', 'org:founded_by',
       'per:religion']

id2label = {idx: label for idx, label in enumerate(class_names)}
label2id = {label: idx for idx, label in enumerate(class_names)}

dataset = pd.read_csv("../dataset/train/train.csv")
label = label_to_num(dataset['label'].values)

dataset['id'] = dataset['id'].astype(str)
dataset['subject_entity'] = dataset['subject_entity'].apply(lambda x : eval(x))
dataset['object_entity'] = dataset['object_entity'].apply(lambda x : eval(x))
dataset['label'] = dataset['label'].map(label2id)

with open('train_eval_idx.pkl', 'rb') as f:
    train_eval_idx = pickle.load(f)

train_dataset = dataset.iloc[train_eval_idx['train']]
dev_dataset = dataset.iloc[train_eval_idx['eval']]

train_label = np.array(label)[train_eval_idx['train']]
dev_label = np.array(label)[train_eval_idx['eval']]

from datasets import Dataset, DatasetDict, Features, Value, ClassLabel

class_names = ['no_relation', 'org:top_members/employees', 'org:members',
       'org:product', 'per:title', 'org:alternate_names',
       'per:employee_of', 'org:place_of_headquarters', 'per:product',
       'org:number_of_employees/members', 'per:children',
       'per:place_of_residence', 'per:alternate_names',
       'per:other_family', 'per:colleagues', 'per:origin', 'per:siblings',
       'per:spouse', 'org:founded', 'org:political/religious_affiliation',
       'org:member_of', 'per:parents', 'org:dissolved',
       'per:schools_attended', 'per:date_of_death', 'per:date_of_birth',
       'per:place_of_birth', 'per:place_of_death', 'org:founded_by',
       'per:religion']

class_labels = ClassLabel(num_classes=30, names=class_names)

hf_train_dataset = Dataset.from_pandas(train_dataset)
hf_dev_dataset = Dataset.from_pandas(dev_dataset)

hf_train_dataset = hf_train_dataset.remove_columns('__index_level_0__')
hf_dev_dataset = hf_dev_dataset.remove_columns('__index_level_0__')

klue_re = DatasetDict({'train' : hf_train_dataset, 'dev' : hf_dev_dataset})
```
